internal/ci/runner.go

```diff
@@ -70,17 +70,21 @@ func (r *Runner) setup(run config.Run) error {
 	log.Info().Msg("Run entry.sh")
 	var out bytes.Buffer
 	var homeDir = utils.GetProjectDir()
-	cmd := exec.Command(homeDir + "/entry.sh", "&")
-	cmd.Stderr = &out
-	if err := cmd.Run(); err != nil {
-		return errors.New("couldn't start test: " + out.String())
+	if os.Getenv("SAUCE_VM") == "" {
+		cmd := exec.Command(homeDir+"/entry.sh", "&")
```

Comment:
It'd be a bit safer to use `path.Join()` here, since from a contractual perspective, `utils.GetProjectDir()` could return a path ending with a slash or without. This will just ensure that it works with both properly.
```suggestion
		cmd := exec.Command(path.Join(homeDir, "/entry.sh"), "&")
```

PS: I deliberately chose `path.Join()` here instead of `filepath.Join()`.